my-profile-webapp/src/main/webapp/my-app.js

```diff
@@ -185,6 +245,24 @@
               });
           }
           
+          var searchUsersLastName = function(searchTerm){          
+            return $http.get('/portal/api/people.json?searchTerms%5B%5D=sn&sn=' + searchTerm).success(
```

Comment:
Yep.  The webservice, portal in this case, handles it's own security logic about who should see which users (funky).  Since you need a portal session, this authenticates you as you and only you and then does the security around some uPortal rules.  I think this is fitting that the webservice is doing it's own security (kind of).  I tried calling the webservice as a non-admin in production and it sends back the correct responses, non-test tiers, portal seems to throw caution to the wind and allow anyone to lookup any user.  Weird, yes.  Funky security, yes.  I agree with your next comment about this not being the biggest deal.  When we get our real service, we'll have the opportunity to use spring security at that point.